[ET][Portable] Check scalar overflow: op_leaky_relu

manuelcandales opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #12027
* #12029
* #12028
* #12026
* #12025
* #12015
* #12014

Differential Revision: [D77401091](https://our.internmc.facebook.com/intern/diff/D77401091/)

manuelcandales committed:
[ET][Portable] Check scalar overflow: op_leaky_relu



Differential Revision: [D77401091](https://our.internmc.facebook.com/intern/diff/D77401091/)

[ghstack-poisoned]

swolchok commented on kernels/test/op_leaky_relu_test.cpp:
```diff
@@ -40,10 +40,29 @@ class OpLeakyReluTest : public OperatorTest {
 #undef TEST_ENTRY
 }
+
+TEST_F(OpLeakyReluTest, FloatTensorTooSmallScalarDies) {
```
wasn't there shared infrastructure for this?

manuelcandales replied:
leaky_relu is different than the others, it only accepts floating point scalars. The shared macro tests integrat dtypes as well, so I cannot use it here.

manuelcandales committed:
Update on "[ET][Portable] Check scalar overflow: op_leaky_relu"


Differential Revision: [D77401091](https://our.internmc.facebook.com/intern/diff/D77401091/)

[ghstack-poisoned]

manuelcandales committed:
Update on "[ET][Portable] Check scalar overflow: op_leaky_relu"


Differential Revision: [D77401091](https://our.internmc.facebook.com/intern/diff/D77401091/)

[ghstack-poisoned]
